gcp/api/server.py

```diff
@@ -200,6 +201,7 @@ def query_by_commit(commit, to_response=bug_to_response):
 def _is_semver_affected(affected_packages, package_name, ecosystem, purl,
                         version):
   """Returns whether or not the given version is within an affected SEMVER
+
```

Comment:
Odd formatting here with the new line. I had this issue before when running `pyformat` to format files, `yapf` seems to not have the same problem for me.

Reply:
I'll get to the bottom of this at some point, it's only cosmetic...